tests/acceptance/AdvancedSettingsCest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class AdvancedSettingsCest {
+  function toggleAnonymousDataSetting(\AcceptanceTester $I) {
+    $I->wantTo('Confirm anonymous data settings can be toggled on Advanced Settings Page');
+    $noAnonymousData = '[data-automation-id="analytics-no"]';
+    $yesAnonymousData = '[data-automation-id="analytics-yes"]';
+    $submitButton = '[data-automation-id="settings-submit-button"]';
+    $successMessage = "Settings saved";
+    $I->login();
+    $I->amOnMailPoetPage('Settings');
+    $I->click('[data-automation-id="settings-advanced-tab"]');
+    $I->waitForText('Bounce email address');
+    $I->selectOption($noAnonymousData, 'No');
+    //save + refresh
+    $I->click($submitButton);
+    $I->waitForText($successMessage);
+    $I->seeOptionIsSelected($noAnonymousData, 'No');
+    //repeat for Yes
+    $I->selectOption($yesAnonymousData, 'Yes');
+    $I->click($submitButton);
+    $I->waitForText($successMessage);
+    $I->seeOptionIsSelected($yesAnonymousData, 'Yes');
+  }
```

Comment:
There should be an empty line between methods. We need to add linters for this though.